### PR TITLE
Add human_reference_atlas subset

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

**Comment by matentzn** [comment]
I cant judge this! You are saying "subset", but you created a component.. I also don't see the related changes to the Makefile!

**Comment by dosumis** [comment]
@matentzn wrote:
> I cant judge this! You are saying "subset", but you created a component.. I also don't see the related changes to the Makefile!

This file is created from a robot template via a build job on https://github.com/hubmapconsortium/ccf-validation-tools. (template is not cached).  It contains only subset and present_in_taxon axioms (to Homo sapiens).  I think it makes sense as a component as it needs to be present in the editors file and available for all release files.  RE:Makefile - i thought it would be sufficient to have an import statement on the editor's file, or does the component need to be explicitly named in the Makefile?

**Review by dosumis — APPROVED** [review]
I'm happy with it if @matentzn is.  Please also see https://github.com/obophenotype/uberon/pull/2346#pullrequestreview-917601946 - which does the same for Uberon.

**Comment by matentzn** [comment]
Same concerns as uberon, otherwise fine

**Comment by dosumis** [comment]
Looking good.  Is this ready to merge?

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ name: Docs
 on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
+  push:
```
Comment (matentzn) [inline .github/workflows/docs.yml:8]: dont commit

**Review by matentzn — APPROVED** [review]
Just dont commit the docs.yml file, else its fine

File: .github/workflows/docs.yml (hunk 1 above)
Comment (anitacaron) [reply .github/workflows/docs.yml]: done

**Pull request merged**